--- pymssql-2.0.0.orig/setup.py	2014-04-07 18:23:53.428169908 -0500
+++ pymssql-2.0.0/setup.py	2014-04-07 18:27:55.029167286 -0500
@@ -137,16 +137,8 @@
     if sys.platform == 'darwin':
         FREETDS = osp.join(ROOT, 'freetds', 'darwin_%s' % BITNESS)
         print("""setup.py: Detected Darwin/Mac OS X.
-    You can install FreeTDS with Homebrew or MacPorts, or by downloading
-    and compiling it yourself.
-
-    Homebrew (http://brew.sh/)
-    --------------------------
-    brew install freetds
-
-    MacPorts (http://www.macports.org/)
-    -----------------------------------
-    sudo port install freetds
+    We'll try using the FreeTDS distributed with the stack since
+    pymssql doesn't currently bundle freetds for darwin.
         """)
 
     if not os.getenv('PYMSSQL_DONT_BUILD_WITH_BUNDLED_FREETDS'):
@@ -187,26 +179,11 @@
     )
 
 if sys.platform == 'darwin':
-    fink = '/sw'
-    if osp.exists(fink):
-        add_dir_if_exists(include_dirs, osp.join(fink, 'include'))
-        add_dir_if_exists(library_dirs, osp.join(fink, 'lib'))
-
-    macports = '/opt/local'
-    if osp.exists(macports):
-        # some mac ports paths
-        add_dir_if_exists(
-            include_dirs,
-            osp.join(macports, 'include'),
-            osp.join(macports, 'include/freetds'),
-            osp.join(macports, 'freetds/include')
-        )
-        add_dir_if_exists(
-            library_dirs,
-            osp.join(macports, 'lib'),
-            osp.join(macports, 'lib/freetds'),
-            osp.join(macports, 'freetds/lib')
-        )
+    #Ignoring fink and macports paths since
+    #these can be used by using the manifest.remap
+    #method.
+    add_dir_if_exists(include_dirs, osp.join(os.getenv('FREETDS_DIR'), 'include'))
+    add_dir_if_exists(library_dirs, osp.join(os.getenv('FREETDS_DIR'), 'lib'))
 
 if sys.platform != 'win32':
     # Windows uses a different piece of code to detect these
